### PR TITLE
Improvements to rewrite for the commit-first-word-as-subject tenet

### PR DESCRIPTION
**Pull request by mullikine**
public == "true" was removed to fix the 'no issues' problem.

This is a test script with many edge-cases.
https://github.com/mullikine/tenets/blob/master/javascript/test-fix-comment.js

### CODE REVIEW

**Comment by Daanikus** [comment]
Yeah, more comments please. And have you got some more examples you can add to the example.go file? Maybe with comments on how they will look afterwards

File: tenets/codelingo/effective-go/comment-first-word-as-subject/codelingo.yaml [hunk 1]
```diff
@@ -13,26 +13,138 @@ funcs:
...
+            if (word.length < 4) return
+            // If start of comment smells like an old name
+            if (lc(word).indexOf(lc(name)) !== -1) {
+              words[0] = name
+            }
+          })
+          return words.join(" ")
+        }
+        
+        var verbs = ["test", "ensure", "replace", "get",
```
Comment (Daanikus) [inline tenets/codelingo/effective-go/comment-first-word-as-subject/codelingo.yaml]:
```suggestion
        var verbs = ["test", "ensure", "replace", "get", "check, "start"
```
